eq_translations/extractable_strings.py

```diff
@@ -136,6 +136,22 @@
         "context": "Question",
         "additional_context": ["ListHeading", "ListDescription"],
     },
+    {
+        "json_path": "$..question.definition.contents[*].title",
```

Comment:
Contents is an array, so each content can have a title, but won't there also be a definition title which is `$..question.definitions[*].title`?. Otherwise it would still be a breaking change.